src/app/drive/components/ShareInviteDialog/ShareInviteDialog.tsx

```diff
@@ -70,158 +70,144 @@ const ShareInviteDialog = (props: ShareInviteDialog): JSX.Element => {
 
   return (
     <div>
-      <div className="flex h-16 w-full items-center justify-between space-x-5 border-b border-gray-10 px-5">
-        <h4
-          className="flex max-w-full items-center space-x-4 overflow-hidden overflow-ellipsis whitespace-nowrap text-xl font-medium"
-          title={translate('modals.shareModal.invite.title')}
-        >
-          <ArrowLeft onClick={props.onInviteUser} size={22} className="cursor-pointer" />
-          <span>{translate('modals.shareModal.invite.title')}</span>
-        </h4>
+      <form className="m flex w-full" onSubmit={handleSubmit(onAddInviteUser)}>
+        <input
```

Comment:
Input used here should match Figma styles, we should use the standard input: `import Input from 'app/shared/components/Input';`

Reply:
I leave this because there is already a related comment and Rafa is on it in this PR https://github.com/internxt/drive-web/pull/721